geowebcache/web/src/main/webapp/WEB-INF/geowebcache-rest-context.xml

```diff
@@ -7,7 +7,6 @@
     http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-4.0.xsd
     http://www.springframework.org/schema/mvc http://www.springframework.org/schema/mvc/spring-mvc-4.0.xsd">
 
-  <mvc:annotation-driven />
```

Comment:
This should be left in. This should be the place where MVC is enabled, since we know it's not used by GeoServer, only GWC standalone.